### PR TITLE
Set default for ToSection topics

### PR DESCRIPTION
**Pull request by disrupted**
allow configuring empty ToSection more easily without having to specify topics attribute, same as FromSection

```yaml
- type: kubernetes-app
  name: example
  from: {}
  to: {}
```

### CODE REVIEW

File: kpops/components/base_components/models/to_section.py [hunk 1]
```diff
@@ -79,20 +79,18 @@ def extra_topic_role(cls, values):
...
-    :type models: dict[str, Any]
     :param topics: Output topics
-    :type topics: dict[str, TopicConfig]
+    :param models: Data models
     """
 
+    topics: dict[TopicName, TopicConfig] = Field(
+        default={}, description=describe_attr("topics", __doc__)
+    )
     # TODO: really multiple models?
```
Comment (sujuka99) [inline kpops/components/base_components/models/to_section.py:89]: Can we address this TODO somehow? I think it is very old.

**Review by sujuka99 — CHANGES_REQUESTED** [review]
Please also update the docs.

[Example](https://bakdata.github.io/kpops/1.4/user/references/components/#__codelineno-1-19)

**Comment by disrupted** [comment]
> Please also update the docs.
> 
> [Example](https://bakdata.github.io/kpops/1.4/user/references/components/#__codelineno-1-19)

do I just have to remove the `required` comment or are there any further changes that you have in mind?

**Comment by sujuka99** [comment]
> > Please also update the docs.
> > [Example](https://bakdata.github.io/kpops/1.4/user/references/components/#__codelineno-1-19)
> 
> do I just have to remove the `required` comment or are there any further changes that you have in mind?

I think it would be nice to also add a comment that tells the user how to define the section as empty.

File: kpops/components/base_components/models/to_section.py (hunk 1 above)
Comment (disrupted) [reply kpops/components/base_components/models/to_section.py:89]: absolutely! will do in a follow up PR :)

File: kpops/components/base_components/models/to_section.py [hunk 2]
```diff
@@ -79,20 +79,18 @@ def extra_topic_role(cls, values):
...
     """Holds multiple output topics
 
-    :param models: Data models
-    :type models: dict[str, Any]
     :param topics: Output topics
-    :type topics: dict[str, TopicConfig]
+    :param models: Data models
     """
 
+    topics: dict[TopicName, TopicConfig] = Field(
```
Comment (raminqaf) [inline kpops/components/base_components/models/to_section.py:86]: Add a snapshot test for this please 🧪

Comment (disrupted) [reply kpops/components/base_components/models/to_section.py:86]: there is no new behavior here, the only thing is that `ToSection` can be instantiated without arguments now, same as `FromSection`

**Comment by disrupted** [comment]
> I think it would be nice to also add a comment that tells the user how to define the section as empty.

isn't that the same as for any other value? you'd just set `to: null`

**Comment by sujuka99** [comment]
> > I think it would be nice to also add a comment that tells the user how to define the section as empty.
> 
> isn't that the same as for any other value? you'd just set `to: null`

Hm, maybe we could say somewhere in the docs that you can set `to: {}` or `to: null` and the difference between the 2? I don't think that this is explained anywhere and it may not be intuitive for new users. Same for the other sections that behave the same.

**Comment by disrupted** [comment]
> Hm, maybe we could say somewhere in the docs that you can set `to: {}` or `to: null` and the difference between the 2? I don't think that this is explained anywhere and it may not be intuitive for new users. Same for the other sections that behave the same.

this should be part of the [defaults documentation page](https://bakdata.github.io/kpops/1.4/user/references/defaults/) that explains how the enrichment works. It's more of a broader effort, since this applies to any pipeline key.

**Review by sujuka99 — APPROVED** [review]
LGTM :)

**Review by raminqaf — APPROVED** [review]

**Pull request merged**